src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs

```diff
@@ -319,6 +313,12 @@ public void LoggerProviderException(string methodName, string ex)
             this.WriteEvent(50, methodName, ex);
         }
 
+        [Event(51, Message = "Skip processing log state of type '{0}'.", Level = EventLevel.Warning)]
```

Comment:
Consider a message which is user-actionable. For example, explain that this state is skipped as it does not implement IROL/IEnum etc. And recommend users to use something that implements IROL/IEnum etc. to fix the issue.